app.py

```diff
@@ -1,18 +1,69 @@
-"""main app file for modeler"""
+#"""main app file for modeler"""
 def main():
     # just leaving these here for future reference (elmiko)
     # from pyspark import sql as pysql
     # spark = pysql.SparkSession.builder.appName("JiminyMod").getOrCreate()
     # sc = spark.sparkContext
-    while True:
+
+    import psycopg2
+    import pyspark
+    from pyspark.sql import SQLContext
+    from pyspark import SparkContext, SparkConf
+    import math
+
+    import modeller
+    import storage
```

Comment:
these imports should all be at the top level. i had left them in the function context just as a reminder, but we should move them to the top. it won't make a huge difference in this file but it's good practice for future python coding as a function that uses an `import` will attempt to import that package any time it is called.